### DISCUSSION TITLE
Node Communication

### DISCUSSION THREAD

**Discussion by cedgman** [Security-Onion-Solutions/securityonion#11019] [2.4]
We are running Security Onion 2.4 RC2 in a distributed deployment and 
we are not seeing data from our remote sensors.

In the documentation for 2.4 it specifies the following firewall ports be open.

All nodes to manager:

    22 (only needed for initial setup)
    5000 (Docker registry)
    8086 (influxdb)
    4505 (Salt)
    4506 (Salt)
    5644 (Elastic Agent)
    443 (Sensoroni)

But when you look at the iptables entries for the sensors there is also

    5044
    5055
    8220
    8443

Does the documentation need to be updated?

We are seeing firewall denies for port 8220 going to the master, but none for
the other ports.

Thanks.

**Comment by dougburks** [comment]
I've updated the page:
https://docs.securityonion.net/en/2.4/firewall.html#node-communication

**Comment by m0duspwnens** [comment]
The firewall denies that you are seeing to the manager, is that for the sensor? If you run `iptables -nL | grep <sensor_IP>` on the manager, do you see it listed?

**Comment by cedgman** [comment]
Sorry I should have been more specific, the firewall denies are from our datacenter firewall and not from the Security Onion manager.
